rpyc/rpyc_slave/FileMonitorService.py

```diff
@@ -0,0 +1,33 @@
+import rpyc
+import os
+import time
+from threading import Thread
+
+
+# The fact that we inherit from rpyc.SlaveService makes the exposed_ prefix not to work.
+# i.e., if we'd inherit from rpyc.Service using the exposed_ prefix would have worked.
+class FileMonitorService(rpyc.SlaveService):
+    class FileMonitor(object):
+        def __init__(self, filename, callback, interval=1):
+            self.filename = filename
+            self.interval = interval
+            self.last_stat = None
+            self.callback = rpyc.async_(callback)  # create an async callback
```

Comment:
This is from the rpyc tutorial. Make the callback function asynchronous, because FileMonitor don't have to wait on the result, just notify on the change.